src/server/controllers/index.js

```diff
@@ -0,0 +1,84 @@
+const  express = require('express');
+var router = express.Router();
+
+/* Require model */
+var models = require('../models');
+/* Require middlewares functions */
+
+router.get('/', function (req, res) {
+    res.render('index.html', {title: "Interview Coding Challenge"});
+});
+
+router.get('/getAllFavStuffs', function (req, res) {
+    models.favorite_stuff.find({}, function (err, data) {
+        if (err) {
+            res.json({error: true, message: "There isn't information yet.", response: ""});
+        }
+        res.json({error: false, message: "All Favorites Stuff", response: data});
+    });
+});
+
+router.get('/getAllFavStuffsNames', function (req, res) {
+    models.favorite_stuff.find({}, {fav_stuff_name: 1}, function (err, data) {
+        if (err) {
+            res.json({error: true, message: "There isn't information yet.", response: ""});
+        }
+        res.json(data);
+    });
+});
+
+router.post('/addNewFavStuff*', function (req, res) {
+    if ((typeof req.body.fav_stuff_name == "undefined") || (typeof req.body.fav_stuff_list == "undefined")) {
```

Comment:
I've already saw this validation, I forgot to add the string and array validations when I send for review. :'(